linux_os/guide/system/accounts/accounts-session/accounts_user_interactive_home_directory_defined/rule.yml

```diff
@@ -8,6 +8,10 @@ description: |-
     Assign home directories to all interactive users that currently do not
     have a home directory assigned.
 
+    This rule checks if the home directory is properly defined in a folder which has
+    at least one parent folder, like "user" in "/home/user" or "/remote/users/user".
+    Therefore, it will report finds for home directories like /users, /tmp or /.
```

Comment:
```suggestion
    Therefore, this rule will report a finding for home directories like <tt>/users</tt>, <tt>/tmp</tt> or <tt>/</tt>,
```

Reply:
Done. Only changed the last "," by a "." : )